src/Controller/Admin/EventController.php

```diff
@@ -23,18 +24,25 @@ class EventController extends AbstractRestController implements ClassResourceInt
      */
     private $repository;
 
+    /**
+     * @var MediaManagerInterface
+     */
+    private $mediaManager;
+
     /**
      * @var DoctrineListRepresentationFactory
      */
     private $doctrineListRepresentationFactory;
 
     public function __construct(
         EventRepository $repository,
+        MediaManagerInterface $mediaManager,
```

Comment:
MediaManager is overloaded and should be avoided for referencing `$mediaRepository` or `$entityManager->getReference` should be used.

Reply:
fixed